accord-core/src/main/java/accord/local/Commands.java

```diff
@@ -1006,13 +1026,14 @@ public void accept(SafeCommandStore safeStore)
                         {
```

Comment:
I personally find names like depender/dependee really confusing, and also generally prefer short identifiers for complex loops so as not to pollute every line. But we can discuss alternative approaches.

There's no distinction between full/partial participants, it's only to prevent passing a non-participating homeKey. The whole concept is bad and will be removed, so don't get too caught up on it. Just read `Unseekables`. But participants (lower case) is a good name, as it is those participating keys for our purposes.

For our purposes here, those participants are only those we share with the prev transaction in the stack, which is our route to the root transaction we're trying to execute. The same goes for the `executeAt` - we don't care about any interaction we have beforehand (e.g., `prev` may execute in a later epoch and so the command store may have lost ownership of a key `cur` participates in, and that we no longer care about)